spec/fixtures/fits_0.8.5_jp2.xml

```diff
@@ -0,0 +1,51 @@
+<?xml version="1.0" encoding="UTF-8"?>

```

Comment:
Does this PR handle both FITS 0.6 and 0.8 equally well? Because that'd be super cool.